Source/WebCore/bindings/js/ScriptModuleLoader.cpp

```diff
@@ -82,7 +82,7 @@ ScriptModuleLoader::~ScriptModuleLoader()
 
 UniqueRef<ScriptModuleLoader> ScriptModuleLoader::shadowRealmLoader(JSC::JSGlobalObject* realmGlobal) const
 {
-    auto loader = makeUniqueRef<ScriptModuleLoader>(m_context, m_ownerType);
+    auto loader = makeUniqueRef<ScriptModuleLoader>(*m_context, m_ownerType);
```

Comment:
We need to first store m_context in Ref/RefPtr before using it.
So do: Ref { *m_context }